.travis.yml

```diff
@@ -1,6 +1,17 @@
 language: python
 python:
   - "2.7"
+env:
```

Comment:
Can you make an explicit unit test env? Otherwise will they end up running at all here?